### DISCUSSION TITLE
Font is not correct

### DISCUSSION THREAD

**Discussion by jaka1-newbie** [dlemstra/Magick.NET#1010] [Help]
```
	MagickReadSettings readSettings1 = new MagickReadSettings{ // Свойства текста
				
				FillColor = new MagickColor(project.Variables["textColor"].Value), // цвет текста
				BackgroundColor = MagickColors.Transparent, // фон текста
				TextGravity = Gravity.Center,
				Font = "Tinsel-&-Glitter-Silhouette-Font-Regular", // Шрифт текста (только те, что установлены в Windows)
				Width = 1400, // Ширина текста
				Height = 1000 }; // Высота текста
				image.Alpha(AlphaOption.Opaque);
			using (MagickImage label = new MagickImage("caption: THE LOST PRINCE", readSettings1))
				{
					image.Composite(label, Gravity.Northwest, 0, 0, CompositeOperator.Over);
				}
```
Font installed, but at finish not used in Magick.Net
[Tinsel-Glitter-5291123.zip](https://github.com/dlemstra/Magick.NET/files/7016663/Tinsel-Glitter-5291123.zip)

**Comment by dlemstra** [comment]
What happens when you set the `Font` to the filename of the font?

**Comment by jaka1-newbie** [comment]
It Work!
Wow

**Comment by jaka1-newbie** [comment]
What the problem? Why?

**Comment by dlemstra** [comment]
It is most likely that you were not using the exact correct name that we would need. You could use the `MagickNET.FontNames` the get all the known font names.

**Reply by jaka1-newbie** [reply]
I receive them by ImageMagick
`convert -list font `
Will try MagickNET.FontNames

**Reply by dlemstra** [reply]
You might also need to remove `-Regular` part from `Tinsel-&-Glitter-Silhouette-Font-Regular`. I use font file names most of the time when I need to use an exotic font.
